packages/react/etc/react.api.md

```diff
@@ -3793,6 +3796,14 @@ export interface IDetailsList extends IList {
 export interface IDetailsListCheckboxProps extends IDetailsCheckboxProps {
 }
 
+// @public (undocumented)
```

Comment:
These should only be callbacks, not 'active' values. We don't want a state where this gets constantly set and unset as the user performs edit operations.

Ideally, just take a callback for `onEditColumn` or something, and that callback should be passed a callback (supplied by `DetailsList`) which will be used to 'commit' the update. That way the caller need only concern itself with showing the dialog and then posting a final update. `DetailsList` already has internal 'commit' flows for resizing and reordering columns, so those should be used.